### PR TITLE
Sign up form

### PR DESCRIPTION
**Pull request by LizzyKate**
Added the signup form

### CODE REVIEW

File: packages/frontend/src/components/SignUp.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { EyeInvisibleOutlined, EyeTwoTone } from "@ant-design/icons";
+import { Form, Input, Button, Row } from "antd";
+
+const SignUp = () => {
+  const [passwordVisible, setPasswordVisible] = useState(false);
+
+  return (
+    <Row justify="center" style={{ marginTop: "32px" }}>
```
Comment (bemijonathan) [inline packages/frontend/src/components/SignUp.tsx:9]: we should avoid inline styling create a layout called authlayout that will  carry both signin signup and update password

File: packages/frontend/src/components/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <Form layout="vertical" style={{ width: "50%" }}>
```
Comment (bemijonathan) [inline packages/frontend/src/components/SignUp.tsx:10]: you have Row  you can do <Col span={12} >

File: packages/frontend/src/components/SignUp.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        </Form.Item>
+        <Form.Item label="Button">
+          <Button>Button</Button>
+        </Form.Item>
+      </Form>
+    </Row>
+  );
+};
+
+export default SignUp;
```
Comment (bemijonathan) [inline packages/frontend/src/components/SignUp.tsx:33]: remove export default, lets use export const

File: packages/frontend/src/components/SignUp.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  const [passwordVisible, setPasswordVisible] = useState(false);
+
+  return (
+    <Row justify="center" style={{ marginTop: "32px" }}>
+      <Form layout="vertical" style={{ width: "50%" }}>
+        <Form.Item label="Name">
+          <Input type="text" placeholder="Enter your full name" />
+        </Form.Item>
+        <Form.Item label="Email">
+          <Input type="email" placeholder="Enter your email" />
```
Comment (bemijonathan) [inline packages/frontend/src/components/SignUp.tsx:15]: lets add validation

**Comment by bemijonathan** [comment]
nice work can we  use descriptive branch name instead of just front end

File: packages/frontend/src/components/SignUp.tsx (hunk 1 above)
Comment (LizzyKate) [reply packages/frontend/src/components/SignUp.tsx:9]: Using Ant design margin-top failed to work
Suggest a better way

**Pull request closed without merging**